### PR TITLE
Added command to view all owned tags in summary and individually

### PR DESCRIPTION
**Pull request by smathep**
Added
!tags claimed <user>
command to view tags owned by a user as suggested in issue #345

### CODE REVIEW

File: ClemBot.Bot/bot/cogs/tags_cog.py [hunk 1]
```diff
@@ -68,6 +68,45 @@ async def tag(self, ctx: commands.Context, tag_name: Optional[str] = None):
...
+            return
+
+        tags_url = f'{bot_secrets.secrets.site_url}dashboard/{ctx.guild.id}/tags'
+        pages = self.chunked_tags(ownedTags, TAG_CHUNK_SIZE, ctx.prefix, 'Available Tags', tags_url)
+        for tag in ownedTags:
+            embed = discord.Embed(color=Colors.ClemsonOrange, title=f'{tag.name}')
+            embed.set_author(name=f'{self.bot.user.name} - Tags', url=LINK_URL, icon_url=self.bot.user.display_avatar.url)
+            embed.add_field(name='Content', value=tag.content)
+            embed.add_field(name='Uses', value=f'{tag.use_count} use{"s" if tag.use_count != 1 else ""}')
+            pages.append(embed)
```
Comment (Jay-Madden) [inline ClemBot.Bot/bot/cogs/tags_cog.py:103]: pages is already an array of embeds

Comment (smathep) [reply ClemBot.Bot/bot/cogs/tags_cog.py:103]: Yes before that loop, it is the summary view of all tags owned by a user. This loop then appends the tag contents as pages so the user can easily see the content of all their tags

Comment (Jay-Madden) [reply ClemBot.Bot/bot/cogs/tags_cog.py:103]: hmm, i would prefer we use the site for that but i can see the appeal

Comment (smathep) [reply ClemBot.Bot/bot/cogs/tags_cog.py:103]: More steps for users

File: ClemBot.Bot/bot/cogs/tags_cog.py [hunk 2]
```diff
@@ -312,6 +351,5 @@ def chunked_tags(self, tags_list: list, n: int, prefix: str, title: str, url: st
 
         return pages
```
Comment (Jay-Madden) [inline ClemBot.Bot/bot/cogs/tags_cog.py:353]:
```suggestion


```

File: ClemBot.Bot/bot/cogs/tags_cog.py (hunk 1 above)
Comment (Jay-Madden) [reply ClemBot.Bot/bot/cogs/tags_cog.py:103]: we ultimately want to push people to the site as that is where new features will be

**Review by Jay-Madden — APPROVED** [review]
lgtm

**Pull request merged**